src/raftstore/store/snap.rs

```diff
@@ -1093,6 +1100,10 @@ impl SnapManager {
                 snap_size: Arc::new(RwLock::new(0)),
             })),
             ch: ch,
+            limiter: Arc::new(RateLimiter::new(
+                10 * 1024 * 1024, // bytes_per_sec
```

Comment:
use config here

Reply:
I will attend to it later, when testing.

Reply:
By far I still can't pass compilation, not very familiar with rust.